notebooks/Liquidators_data.ipynb

```diff
@@ -0,0 +1,977 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2cc808a8",
+   "metadata": {},
+   "source": [
+    "# Analyze the behavior of liquidators.\n",
+    "\n",
+    "Steps:\n",
+    "1. Load a sample of events from the Google Storage for each of the following protocols: \"zklend\", \"nostra_alpha\", \"nostra_mainnet\", \"hashstack_v0\", and \"hashstack_v1\". You can find the sample e.g. for zkLend here: https://storage.googleapis.com/derisk-persistent-state/zklend_data/events_sample.parquet. To obtain the sample for other protocols, just change the folder in the address, e.g. https://storage.googleapis.com/derisk-persistent-state/nostra_alpha_data/events_sample.parquet for Nostra Alpha. Write the loading part in a way that the source can be easily changed from the Google Storage to a local database.\n",
+    "2. Identify only liquidation events. Use Starkscan to explore their structure. For example, this is one of zkLend's liquidation events. https://starkscan.co/event/0x00a1e829fb091f7493c509135992ac12af6a81177e3ffab5c34951fb29c23084_6\n",
+    "3. Identify the amounts of debt tokens liquidated.\n",
+    "4. Visualize the behavior of all liquidators found in the sample in a Jupyter notebook. You should use the visualizations to answer the following questions:\n",
+    "\n",
+    "How many liquidators are there per protocol? Do some of them operate on more than just a single lending protocol?\n",
+    "\n",
+    "How many loans has each of them liquidated?\n",
+    "\n",
+    "What is the average liquidated debt amount? Compute the averages across all liquidations, but also per protocol and per token amounts.\n",
+    "\n",
+    "What is the total amount of the liquidated debt? Compute the total amounts across all liquidations, but also per protocol and per token amounts.\n",
+    "\n",
+    "Fetch prices for these tokens (in USD), e.g., from Coingecko. Multiply the token amounts by the corresponding prices to obtain USD nominal amounts and be able to compare the per token stats across tokens."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 167,
+   "id": "bf107fa7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: psycopg2 in c:\\users\\molak\\anaconda3\\lib\\site-packages (2.9.9)\n",
+      "env: PG_CONNECTION_STRING=''  # nostra_mainnet.py has dependencies. To avoid error adding dummy env\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install psycopg2\n",
+    "import requests\n",
```

Comment:
Organize all import into blocks (standard library imports, 3rd party imports, local (e.g. src.nostra_alpha) imports) separated by empty lines. Also, pls sort them.

Reply:
Pls sort the imports (alphabetically).